test/metabase/pulse/render/js_svg_test.clj

```diff
@@ -91,143 +91,162 @@
   (and (vector? x) (= (first x) "#text")))
 
 (defn- combo-chart-string
-  [series settings]
+  [series-seqs settings-seqs]
```

Comment:
changes in this file are mainly from the API change from `js-svg/combo-chart` which now expects a vector of series and a vector of settings instead.

So I just wrap all `series` and `settings` in a vector and rename them.